.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: php
+
+dist: trusty
+
+env:
+  global:
+    - COMPOSER_ROOT_VERSION=2.x-dev
+
+matrix:
+  include:
+    - php: 5.6
+      env: PHPUNIT_TEST=core,cwpcore
+    - php: 7.0
+      env: PHPUNIT_TEST=admin,cwpcms,Default
+    - php: 7.1
+      env: PHPCS_TEST=1 PHPUNIT_COVERAGE_TEST=1
+    - php: 7.2
+      env: PHPUNIT_TEST=cwpsearch
+
+before_script:
+  # Init PHP
+  - phpenv rehash
+  - phpenv config-rm xdebug.ini
+
+  # Install composer dependencies
+  - composer validate
+  - composer install --prefer-dist --no-interaction --no-progress --no-suggest --optimize-autoloader --verbose --profile
+
+script:
+  - if [[ $PHPUNIT_TEST ]]; then vendor/bin/phpunit --testsuite ${PHPUNIT_TEST/1/Default}; fi
```

Comment:
I _still_ don't understand this `--testsuite` line. Can we make it simple please? `--testsuite $TESTSUITE` and define `TESTSUITE=foo,bar` in env vars

Reply:
Cool. Sorry I didn't click that the shell variable syntax was the bit you were objecting to.

I'll change it but explain for reference anyway, it's a simple `VALUE/search/replace` syntax, similar to that of many a regex implementation (e.g. PCRE, Vi, etc). Im reasonably sure it's POSIX defined, and certainly supported by `bash`.

When I originally put this in to test, all the tests were still defined as `PHPUNIT_TEST=1`